### ISSUE TITLE
can't parse response: OFX SECURITY header not NONE

### ISSUE THREAD

**Issue by davidalpert** [aclindsa/ofxgo#39]
Parsing OFX responses with `SECURITY` setting of `TYPE1` fails with a validation error:
```
can't parse response: OFX SECURITY header not NONE
```

My bank still uses OFX 1.02 OFXSGML format, but uses `SECURITY:TYPE1`:
```
OFXHEADER:100
DATA:OFXSGML
VERSION:102
SECURITY:TYPE1
ENCODING:USASCII
CHARSET:1252
COMPRESSION:NONE
OLDFILEUID:NONE
NEWFILEUID:NONE
```

This is parsed beautifully after I change that security setting to `NONE`:
```
OFXHEADER:100
DATA:OFXSGML
VERSION:102
SECURITY:NONE
ENCODING:USASCII
CHARSET:1252
COMPRESSION:NONE
OLDFILEUID:NONE
NEWFILEUID:NONE
```

Apparently, this is a common issue in financial software processing `.ofx` responses, with the "fix" for users to hand-edit the responses to remove the `TYPE1` security value and replace it with `NONE`:
- e.g. [Statement Import Error (OFX Security Unsupported)](https://support.cosmolex.com/knowledge-base/statement-import-error-ofx-security-unsupported/)
 
When I went digging to learn more about this setting I found [Open Financial Exchange Specification 2.2, Nov 26, 2017](https://www.ofx.net/downloads/OFX%202.2.pdf) which states in section `4.2.2.2 Type 1 Protocol Overview` that:

```
    Type 1 applies only to the request part of a message; the server response is unaffected.
```

Thus I think we could remove [this validation](- https://github.com/aclindsa/ofxgo/blob/2b8a79e4b74b0ca32c2b6f21147841d22c656dca/response.go#L95-L98
) as `TYPE1` security doesn't seem to impact responses.

**Comment by aclindsa** [comment]
Interesting! Thanks for bringing this to my attention.

I think I would prefer to relax the validation rather than removing it completely (i.e. explicitly allow *either* "NONE" or "TYPE1", rather than allowing any possible value). Is this something you're willing and able to provide a patch for?

**Comment by davidalpert** [comment]
for sure! patch coming up.

**Comment by davidalpert** [comment]
please see PR #40

**Comment by aclindsa** [comment]
Thanks for fixing this up for me and even providing an additional test case!

**Comment by davidalpert** [comment]
@aclindsa would it be possible to release a patch version with this fix in it? say `0.1.2`?

the latest release appears to be `0.1.1` and [does not yet contain](https://github.com/aclindsa/ofxgo/compare/v0.1.1...master#diff-6b94feb58536b61f1aaf93471d52ee58748d9252f08e12cd9cce2feb44c49691R96) the ability to parse `SECURITY` value of `TYPE1`.

**Comment by aclindsa** [comment]
> @aclindsa would it be possible to release a patch version with this fix in it? say `0.1.2`?
> 
> the latest release appears to be `0.1.1` and [does not yet contain](https://github.com/aclindsa/ofxgo/compare/v0.1.1...master#diff-6b94feb58536b61f1aaf93471d52ee58748d9252f08e12cd9cce2feb44c49691R96) the ability to parse `SECURITY` value of `TYPE1`.

Absolutely, `0.1.2` is tagged now!

**Comment by davidalpert** [comment]
thank you so much! 👏
